server/src/test/java/org/folio/metastorage/server/MainVerticleTest.java

```diff
@@ -2287,10 +2296,29 @@ public void testOaiSimple() throws XMLStreamException, IOException, SAXException
         .contentType("text/plain")
         .body(containsString("Error"));
 
-    //PUT disable the transformer
+    // OAI config with unknown transformer
+    JsonObject oaiConfigBadTransformer = new JsonObject()
+        .put("transformer", "doesnotexist");
+    RestAssured.given()
+        .header(XOkapiHeaders.TENANT, tenant1)
+        .header("Content-Type", "application/json")
+        .body(oaiConfigBadTransformer.encode())
+        .put("/meta-storage/config/oai")
+        .then()
+        .statusCode(204);
 
-    JsonObject oaiConfigOff = new JsonObject()
-      .put("transformer", "");;
+    RestAssured.given()
+        .header(XOkapiHeaders.TENANT, tenant1)
+        .param("set", "issn")
+        .param("verb", "ListRecords")
+        .param("metadataPrefix", "marcxml")
+        .get("/meta-storage/oai")
+        .then().statusCode(400)
```

Comment:
This should be a 500 since it's server misconfoguration

Reply:
All future failures default to 400.  This error is caused by user behavior.  Not an internal failure

Reply:
Have it your way!